### PR TITLE
Interposed questions

### PR DESCRIPTION
**Pull request by bastianjoel**
Requires https://github.com/OpenSlides/openslides-autoupdate-service/pull/814 and https://github.com/OpenSlides/openslides-backend/pull/2103

Marking a speak as intervention is currently not possible because of a backend bug.

### CODE REVIEW

File: client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts [hunk 1]
```diff
@@ -480,6 +532,46 @@ export class ListOfSpeakersContentComponent extends BaseMeetingComponent impleme
...
+                : (end - (speaker.begin_time + total_pause + default_time)) * -1;
+            return {
+                running: speaker.isSpeaking,
+                default_time,
+                countdown_time: speaker.begin_time ? countdown_time : default_time
+            };
+        }
+
+        return null;
+    }
```
Comment (luisa-beerboom) [inline client/src/app/site/pages/meetings/modules/list-of-speakers-content/components/list-of-speakers-content/list-of-speakers-content.component.ts:573]: This function is a bit long, maybe split it up?

**Review by luisa-beerboom — APPROVED** [review]
Generally looks good, but one small remark from me

**Review by Elblinator — CHANGES_REQUESTED** [review]
please fix:
- [x] move the tool tips above the play/stop/pause button
- [x] display the correct icon if the intervention is in the list (currently displayed icon -> "help")
- [x] change the order of the user  information. currently the time of the countdown is on the left side, change the displayed time to be the closed to the play/stop/pause buttons
- [x] the structure level name is missing in the "Last speakers"-list
- [x] change structure level colour to accent colour (white while the background is in accent colour)
- [x] disable interposed question button while no one is talking
- [ ] for interposed questions: disable stop button while  no speaker is selected
- [ ] ~~allow to start the interposed question even while the parent is paused (paused is not the same as stopped)~~ Backend issue
- [x] if the interposed question is started it should be in accent colours and there should be a mic icon
- [x] if a contribution is started -> paused -> stopped, then the pause-time is subtracted from the timer 
- [ ] ~~after selecting a contribution as intervention, an ui issue occurs in the mat-menu, the intervention is not deselectable and other kinds of contributions are not visible~~ wontfix

**Comment by jsangmeister** [comment]
> if a contribution is started -> paused -> stopped, then the pause-time is subtracted from the timer

Should be fixed with https://github.com/OpenSlides/openslides-backend/pull/2103/commits/3e632223ad4f72ce545dd1d69b85a7a31d2c3544

**Review by luisa-beerboom — APPROVED** [review]

**Review by Elblinator — CHANGES_REQUESTED** [review]
After adding a person with one structure level to a empty LoS the following Error is thrown. @bastianjoel, can you please give an assessment if that is an Error from this PR, the backend or from somewhere else

If a structure level and the type intervention is selected, then the timer from the structure level is displayed and changed. the timer from the intervention shiould be used for interventions

**Review by Elblinator — APPROVED** [review]

**Pull request merged**